README.md

```diff
@@ -39,10 +39,27 @@ Prerequisites
   - For macOS : `brew install haskell-stack`
   - For Windows, following the instructions
   [here](https://docs.haskellstack.org/en/stable/install_and_upgrade/#windows).
-  
+
   It is required at least 8GB RAM for `stack` installation.
 
-* You might need to install the following third-party libraries.
+* Install [Nix](https://github.com/NixOS/nix):
+
+  - For Arch Linux
+    + [ArchLinux Nix](https://aur.archlinux.org/packages/archlinux-nix/)
+    + and the nix package `pacman -S nix`
+
+  - Building From Source
+    + https://github.com/NixOS/nix
+
+* And run the following Nix commands:
+
+  ```bash
+  nix-channel --add https://nixos.org/channels/nixpkgs-unstable
+  nix-channel --update
+  nix-env -u
+  ```
+
+* Nix will isntal
```

Comment:
Typo "install"

Reply:
It'll be great to be consistent with `stack` and list the same OS here too, not only arch linux

Reply:
Agreed, however I'm unsure of the exact process on making it work on other OS's

Reply:
I made a PR to merge into this one for the debian setup: #1081